popup.js

```diff
@@ -91,6 +106,9 @@ document.addEventListener('DOMContentLoaded', function () { // this function  st
         // On button click, POST all the field data in trello board
         checkPageButton.addEventListener('click', function () {
 
+            // Button Click Event
+            _gaq.push(['_trackEvent', 'Add To Trello', 'clicked']);
```

Comment:
I believe this code doesn't actually send the event click to the analytics - the official docs use this as a function that is sent as an addEventListener. Let me know if I am misunderstanding this. I cannot see the button click box on the overview page on the Analytics page either.

Reply:
@HarshdipD  Actually According to docs we just have to push the event to gaq global array. Analytics script listens to gaq array changes and fires the event asynchronously in background.

Reply:
Thanks, Bhavesh, just to confirm if I'm getting this right, I don't see the button name on the analytics overview page, as it is supposed to be in docs. Could you provide a screenshot of it working for you so I know where to look?